apps/backend/src/models/Call.ts

```diff
@@ -33,6 +33,16 @@ export class Call {
   ) {}
 }
 
+export class CallHasInstrument {
```

Comment:
It might be worth creating a "CallHasInstrument" model file for this because it represents a different table.